srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -0,0 +1,292 @@
+import argparse
+from subprocess import call, PIPE
+from typing import Dict, Any
+
+import yaml
+from h5py import File as h5open
+import numpy as np
+from os import makedirs, path
+import pandas as pd
+from qcore import binary_version, srf, geo, simulation_structure
+from srf_generation.source_parameter_generation.uncertainties.common import (
+    HF_RUN_PARAMS,
+    BB_RUN_PARAMS,
+    LF_RUN_PARAMS,
+)
+
+from srf_generation.source_parameter_generation.uncertainties.mag_scaling import mag2mom
+
+SRF2STOCH = "srf2stoch"
+GENERICSLIP2SRF = "generic_slip2srf"
+FAULTSEG2GSFDIPDIR = "fault_seg2gsf_dipdir"
+
+
+def gen_stoch(stoch_file, srf_file, single_segment=False):
+    """
+    Creates stoch file from srf file.
+    """
+    out_dir = path.dirname(stoch_file)
+    makedirs(out_dir, exist_ok=True)
+    dx, dy = 2.0, 2.0
+    if not srf.is_ff(srf_file):
+        dx, dy = srf.srf_dxy(srf_file)
```

Comment:
This part trips me out. But I guess that the stoch file for a point source (aka not finite fault) only ends up with one point regardless.

Reply:
This is carried over from the old workflow so *shrug*?

Reply:
Yeah, I checked that before commenting - it seems to work but probably an easy check.